projects/graph/src/graph.py

```diff
@@ -1,9 +1,56 @@
 """
 Simple graph implementation compatible with BokehGraph class.
 """
+from random import random
+
+
+class Vertex:
+    def __init__(self, label):
+        self.label = label
+        self.edges = set()
+        self.x = random() * 5
+        self.y = random() * 5
```

Comment:
Nice use of random() here! It's not necessary for vertex generation, but it will give you something new to look at each time.